htdocs/comm/action/listactions.php

```diff
@@ -241,6 +241,7 @@
 if ($status == '100') { $sql.= " AND a.percent = 100"; }
 if ($status == 'done' || $status == '100') { $sql.= " AND (a.percent = 100 OR (a.percent = -1 AND a.datep2 <= '".$db->idate($now)."'))"; }
 if ($status == 'todo') { $sql.= " AND ((a.percent >= 0 AND a.percent < 100) OR (a.percent = -1 AND a.datep2 > '".$db->idate($now)."'))"; }
+if ($status == 'late') { $sql.= " AND ((a.percent >= 0 AND a.percent < 100 AND a.datep < '".$db->idate($now)."'))"; }
```

Comment:
We can't share this into the status because this will break future change where we can choose several lines in combo on the filters.
I would prefer to get ability to search on the field dateend so we can achieve the goal with filter and date + filter on status.